build/arch.aarch64.mk

```diff
@@ -5,7 +5,17 @@ ELFTYPE := elf64-littleaarch64
 ELFARCH := aarch64
 
 ifeq ($(KERNEL), 1)
+KASAN ?= 0
 CFLAGS += -mcpu=cortex-a53+nofp -march=armv8-a+nofp -mgeneral-regs-only
+ifeq ($(KASAN), 1)
+  # Added to files that are sanitized
+  CFLAGS_KASAN = -fsanitize=kernel-address \
+				 -fasan-shadow-offset=0xe0001f0000000000 \
+                 --param asan-globals=1 \
```

Comment:
Inconsistent indentation of `-fasan-shadow-offset`

Reply:
I have fixed that but I think that something is still broken and git create wrong diff.
Please look at [sources](https://github.com/pj1031999/mimiker/blob/aarch64-kasan/build/arch.aarch64.mk) -- I have added extra spaces.

Reply:
I mean, it's not a big deal :smile: , but I think the problem is that other lines have spaces only, and -`fasan-shadow-offset` has both tabs and spaces
![image](https://user-images.githubusercontent.com/24853486/112749480-4ccb1a00-8fba-11eb-9075-0abf3c18bdae.png)

Reply:
Ok, I retabed that file, but other files in `build` are still broken.
We have `vim: tabstop=8 shiftwidth=8 noexpandtab:` in that files but we are using 2 spaces.
@cahirwpz  could you look at that?